pvlib/pvsystem.py

```diff
@@ -738,7 +739,7 @@ def fuentes_celltemp(self, poa_global, temp_air, wind_speed):
 
         Returns
         -------
-        temperature_cell : Series or tuple of Series
+        numeric or tuple of numeric
```

Comment:
It was a mistake to change the `fuentes_celltemp` docstring that way. `noct_sam_celltemp` should be able to accept `numeric` and return the same type. `numeric` includes `Series`, right?

Reply:
numeric is loosely defined as anything that can be added/multiplied/etc. So that includes `Series`, but also python scalars, numpy arrays of arbitrary dimension, and DataFrames.

Reply:
Err, from [pvlib contributing documentation](https://pvlib-python.readthedocs.io/en/stable/contributing.html#documentation):

> numeric : scalar, np.array, pd.Series. Typically int or float dtype.

also relevant #765

Reply:
That's clear, thanks. The underlying function is tested for `float`, `array` and `Series`. I've added to the method test to cover the types. I think `numeric` is appropriate for this docstring.